config/management/operational/src/command.rs

```diff
@@ -309,8 +322,17 @@ impl Command {
     }
 }
 
+/// A result wrapper for displaying either a correct execution result or an error.
 #[derive(Serialize)]
 pub enum ResultWrapper<T> {
     Result(T),
     Error(String),
 }
+
+/// A struct wrapper for displaying unvalidated transaction contexts.
+#[derive(Serialize)]
+struct UnvalidatedTransactionContext<'a> {
```

Comment:
This is probably a good argument for using an enum instead
```
enum TransactionContext {
  context_only: InnerTransactionContext,
  executed: ExecutedTransactionContext,
}

struct ExecutedTransactionContext {
  result: String,
  inner: InnerTransactionContext,
}
```

Reply:
Thanks David. I'll note this down as a refactor to make in a subsequent PR (so that this PR doesn't block the stream of others)